ci/azure-cross-compile.yml

```diff
@@ -15,6 +15,9 @@ jobs:
       mips:
         vmImage: ubuntu-16.04
         target: mips-unknown-linux-gnu
+      arm:
+        vmImage: ubuntu-16.04
+        target: arm-unknown-linux-gnueabi
```

Comment:
Seems [`arm-unknown-linux-gnueabi` has `AtomicU64`.](https://github.com/rust-lang/rust/blob/4ac4809ccf5f77083ae7155dcc83e921341c2614/src/librustc_target/spec/arm_unknown_linux_gnueabi.rs#L5)

Reply:
@taiki-e Weird, hopefully cfg_target_has_atomic will enable more accurate enabling of features.

My builds were failing when building for a relatively old ARMv5 target.

Reply:
> My builds were failing when building for a relatively old ARMv5 target.

I think it is [`armv5te-unknown-linux-gnueabi`](https://github.com/rust-lang/rust/blob/4ac4809ccf5f77083ae7155dcc83e921341c2614/src/librustc_target/spec/armv5te_unknown_linux_gnueabi.rs#L20).

Reply:
https://github.com/rust-embedded/cross#supported-targets

I only see a armv5te musl target supported,  armv5te-unknown-linux-gnueabi _might_ work, I'm gonna try it out.